pom.xml

```diff
@@ -680,6 +680,15 @@
             </execution>
             -->
           </executions>
+          <configuration>
+            <args>
+              <arg>-unchecked</arg>
+              <arg>-deprecation</arg>
+              <arg>-feature</arg>
+              <arg>-explaintypes</arg>
+              <arg>-Xfatal-warnings</arg>
```

Comment:
There is no way to use `<arg>-Ywarn-unused-import</arg>` at this stage
Because we have the following code
```
// Can Remove this if celeborn don't support scala211 in future
import org.apache.celeborn.common.util.FunctionConverter._
```

Reply:
then we need to update the PR description.

Reply:
updated